app/src/main/java/tech/relaycorp/courier/data/database/DateConverter.kt

```diff
@@ -0,0 +1,12 @@
+package tech.relaycorp.courier.data.database
+
+import androidx.room.TypeConverter
+import java.util.Date
+
+class DateConverter {
+    @TypeConverter
+    fun toDate(dateLong: Long) = Date(dateLong)
```

Comment:
Side note/question: I guess the JVM library should be changed to use `Date` instead, right? We didn't make a decision explicitly yesterday but I guess that seems like the best approach (or the least bad approach :) )

Reply:
Since we don't any particular timezone support, I would vote for using just `Date`. Using an Android specific library on our library, or changing its license don't seem like great options.

Reply:
Agreed. Created task to remind myself to fix it soon: https://github.com/relaycorp/relaynet-jvm/issues/29